.github/workflows/ci.yml

```diff
@@ -131,6 +131,7 @@ jobs:
           '!' -path ./runner '!' -path ./runner-cache -exec rm -rf '{}' ';'
           df -h /
           cd -
+          sed -i -r -e s/Debug/RelWithDebInfo/ build.sh
```

Comment:
What's the `build.sh` script?  Is there a way to directly set the build type?  Because this seems like it could change some other `Debug`s that happen to be in the script.

Reply:
It's https://github.com/immunant/llvm-project/blob/am/mte_instrumentation/build.sh which I didn't want to touch on that branch because it's used by all branches of this project. If this change helps I'll patch the script over there and invalidate the cache on this side.